rust/sdk-processor/src/config/processor_config.rs

```diff
@@ -55,6 +55,28 @@ impl ProcessorConfig {
     pub fn name(&self) -> &'static str {
         self.into()
     }
+
+    /// Get the Vec of table names for parquet processors only.
```

Comment:
nit: add newline after first sentence for nicer [docstring parsing](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components)